src/VCAuthn/Views/Authorize/Authorize.cshtml

```diff
@@ -0,0 +1,79 @@
+@model  VCAuthn.IdentityServer.Endpoints.AuthorizationViewModel
+@{
+    Layout = "_Layout";
+    var qrUri = $"https://chart.googleapis.com/chart?cht=qr&chs=300x300&chld=L|0&chl={Uri.EscapeDataString(Model.Challenge)}";
+}
+<span>
+  <input id="pollInterval" value="@Model.Interval" style="display: none" />
+  <input id="pollUrl" value="@Model.PollUrl" style="display: none" />
+  <input id="resolutionUrl" value="@Model.ResolutionUrl" style="display: none" />
+
+  <div class="container">
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="704.094" height="441" viewBox="0 0 704.094 441">
+        <g id="Symbol_3_9" data-name="Symbol 3 – 9" transform="translate(-158.5 -31)">
+          <g id="Group_28" data-name="Group 28" transform="translate(-1 -8)">
+            <g id="Group_26" data-name="Group 26" transform="translate(112)">
+              <g id="iphone_6:7_black" data-name="iphone 6:7 black" transform="translate(2333.094 909) rotate(90)">
+                <rect id="Rectangle_1" data-name="Rectangle 1" width="325" height="703.094" rx="58" transform="translate(-808 1582)" fill="#fff" stroke="#707070" stroke-width="1"/>
+                <rect id="IMG_4428" width="301.671" height="536.572" rx="4" stroke-width="1" fill="#fff" stroke="#707070" transform="translate(-795.933 1663.25)"/>
+              </g>
+            </g>
+          </g>
+        </g>
+        <image id="frame" width="288" height="288" transform="translate(209.5 76)" xlink:href="@qrUri"/>
+      </svg>
+  </div>
+</span>
+
+<button class="back" onclick="history.back(-1)">
+  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="fff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-arrow-left"><line x1="19" y1="12" x2="5" y2="12"></line><polyline points="12 19 5 12 12 5"></polyline></svg>
+</button>
+
+<style>
+  * {
+    box-sizing: border-box;
+  }
+  
+  body {
+    background: #333132;
+    padding: 0;
+    margin: 0;
+    border:0;
+    height: 100vh;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 12px;
+    height: 100vh;
+    width: 100vw;
+  }
+
+  .back {
+    background: none;
+    border: none;
+    position: absolute;
+    top: 0;
+    left:0;
+    width: 64px;
+    height: 64px;
+  }
+</style>
+
+<script type="text/javascript">
+    $(document).ready((function poll() {
+        setTimeout(function () {
+            $.ajax({
```

Comment:
Do we want a time out?

Reply:
We do, otherwise it'll spam

Reply:
Sorry, I mean a time out after x minutes to stop polling

Reply:
Ah, gotcha! It sounds like a useful practice to prevent an opened page from spamming indefinitely. For the simplicity of this reference implementation, I think we can leave it out.